autoshop/api/v1/serializers.py

```diff
@@ -0,0 +1,9 @@
+from core.models import Profile
+from rest_framework import serializers
+
+
+class ProfileSerializer(serializers.ModelSerializer):
```

Comment:
We might want to rename it to `UserProfileSerializer` for clarity of whose profile its is.